src/components/ContentWidgets/Header/HeaderEditor.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { WidgetEditorProps } from '../../ContentMapping/ContentMapping';
+import { Color, EMPTY_HEADER } from './HeaderConstants';
+import { Header } from './Header';
+import './Header.css';
+
+export const HeaderEditor: React.FC<WidgetEditorProps> = ({
+  editedContent,
+  setEditedContentOnChange
+}) => {
+    const headerContent = editedContent.header_content || EMPTY_HEADER;
+
+    return <> 
+        <div className="header-editor">
+            <div>
+                <p>Header Name </p>
+                <input type="text" 
+                    value={headerContent.header_text}
+                    onChange={(e)=>{
+                        headerContent.header_text = e.target.value;
+                        setEditedContentOnChange("header_content", headerContent)}}>
+                </input>
+            </div>
+            <div>
+                <p>Text Size </p>
+                <input 
+                    type="number"
+                    min="0"
+                    value={headerContent.text_size}
+                    onChange={(e)=>{
+                        headerContent.text_size = Number(e.target.value);
+                        setEditedContentOnChange("header_content", headerContent)}}
+                ></input>
+            </div>
+
+            <div className="header-radio-div">
+                <p>Text Color </p>
+                <label>
+                    <input
+                        type="radio"
+                        value={Color.WHITE}
+                        checked={headerContent.text_color===Color.WHITE}
```

Comment:
nitpick: space between variables and `===`

Reply:
Also `checked` should be based off a boolean, not by checking whether the value is set to white or not.